### ISSUE TITLE
functions should not be hidden inside inline php files and should use correct naming.

### ISSUE THREAD

**Issue by danmarsden** [Behaviour-Analytics/moodle-block_behaviour#7]
https://github.com/Behaviour-Analytics/moodle-block_behaviour/blob/master/update-coords.php#L168

more information on our naming conventions here:
https://docs.moodle.org/dev/Coding_style#Functions_and_Methods

**Comment by danmarsden** [comment]
more in view.php:
https://github.com/Behaviour-Analytics/moodle-block_behaviour/blob/master/view.php#L487

**Comment by danmarsden** [comment]
export.php is another one - you could probably move that function into a locallib.php file like other plugins use (and call it block_behaviour_export_logs())

**Comment by tedkrahn** [comment]
I have removed all functions from inline scripts and placed them into the locallib.php. I have also removed the classes/clusters.php file and integrated it into the scheduled task class, as that is the only place clusters.php is used. Both the export.php and import.php scripts are also integrated into the locallib.php.  All functions are prefixed with block_behaviour, except for classes that are defined in their own namespace such as the scheduled task and event classes.
